### ISSUE TITLE
How to work on this

### ISSUE THREAD

**Issue by chetanbane** [colmena/colmena#242]
Hi,

I am new to this CMS I don't know much about this CMS. 
I am using win7 32 bit

Currently My task to create simple cms for my static sites which we are going to convert in angular-4.

I decided to use this cms 3 days completed neither start admin nor front...

Please guide me on this

INITDB=1 nps dev
nps executing: nps -p dev.admin,dev.api
nps executing: cd admin && nps dev
nps executing: cd api && nps dev
nps executing: ng serve --port 9000 --hmr true -e hmr --host 0.0.0.0 --progress               false --proxy-config proxy.conf.json
nps executing: NODE_ENV=development nodemon server/server.js --watch common --wa              tch server --ext js,json
[nodemon] 1.11.0
[nodemon] to restart at any time, enter `rs`
[nodemon] watching: D:\node-projects\colmena-cms\api\common/**/* D:\node-project              s\colmena-cms\api\server/**/*
[nodemon] starting `node server/server.js`
User: registration enabled
Container does not exist, creating container default
internal/streams/legacy.js:59
      throw er; // Unhandled stream error in pipe.
      ^

Error: getaddrinfo ENOTFOUND images.freeimages.com images.freeimages.com:80
    at errnoException (dns.js:28:10)
    at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:73:26)
[nodemon] app crashed - waiting for file changes before starting...

NOTICE Hot Module Replacement (HMR) is enabled for the dev server.
  The project will still live reload when HMR is enabled,
  but to take advantage of HMR additional application code is required
  (not included in an Angular CLI project by default).
  See https://webpack.github.io/docs/hot-module-replacement.html
  for information on working with HMR for Webpack.
** NG Live Development Server is running on http://0.0.0.0:9000 **
[HPM] Proxy created: /config.js  ->  http://0.0.0.0:3000/config.js
[HPM] Subscribed to http-proxy events:  [ 'error', 'close' ]

[image](https://cloud.githubusercontent.com/assets/3724156/26053778/be54a746-3987-11e7-8cec-3f2092e25148.png)

**Comment by chetanbane** [comment]
Hi I have make some changes & it's work for me now
System Windows 7 32Bit

Please Follow below URL
https://github.com/colmena/colmena-cms

Colmena CMS Change to makes it Work
npm install -g win-node-env

colmena-cms\admin\protractor.conf.js
baseUrl: 'http://0.0.0.0:9000/', >> baseUrl: 'http://localhost:9000/',

colmena-cms\api\config\default.yaml
baseUrl: http://0.0.0.0:3000 >> baseUrl: http://localhost:3000
url: http://0.0.0.0:9000 >> url: http://localhost:9000

colmena-cms\admin\src\lib\lb-sdk\lb.config.ts
private static path: string = '//0.0.0.0:3000'; >> private static path: string = '//localhost:3000';

colmena-cms\admin\proxy.conf.json
"target": "http://0.0.0.0:3000/config.js", >> "target": "http://localhost:3000/config.js",

colmena-cms\api\server\config.json
"host": "0.0.0.0", >> "host": "localhost",

**Comment by beeman** [comment]
@chetanbane the project should now be better usable on Windows.

Please try the latest develop branch. If it does not work, feel free to open a new issue and I'll make sure it works on Windows :)
